### ISSUE TITLE
Cannot find module './lib' in 1.1.6 from npm

### ISSUE THREAD

**Issue by fippo** [BohdanTkachenko/eslint-plugin-require-path-exists#23]
Hi,

my CI server just yelled at me and since you published a new version 18 minutes ago... 
looks like the makefile wasn't run?
```
Cannot find module './lib'
Error: Cannot find module './lib'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/travis/build/opentok/rtcstats/node_modules/eslint-plugin-require-path-exists/index.js:1:80)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
```
Full details in this travis log:
https://travis-ci.org/opentok/rtcstats/jobs/225509507

**Comment by BohdanTkachenko** [comment]
Hi @fippo Thanks for noticing that. Just published fixed version 1.1.7

**Comment by fippo** [comment]
All green, thanks for the quick response!
